### PR TITLE
KAS-3295 Refactor agendaitem properties

### PR DESCRIPTION
**Pull request by sergiofenoll**
Backend PR: https://github.com/kanselarij-vlaanderen/app-kaleidos/pull/267

Remove record & rename explanation to comment

### CODE REVIEW

**Comment by ValenberghsSven** [comment]
`explanation` is still used, missing changes in several files + rename stuff in tests.
Beware that here is an "explanation" used for the search tooltip

**Comment by sergiofenoll** [comment]
> `explanation` is still used, missing changes in several files + rename stuff in tests. Beware that here is an "explanation" used for the search tooltip

I somehow managed to completely forget to do 90% of the work... think I was going on autopilot or something.

Thanks for pointing it out Sven, putting this back in draft until I get that done.

File: app/components/agenda/agendaitem/agendaitem-case/agendaitem-titles.hbs [hunk 1]
```diff
@@ -99,13 +99,13 @@
...
-    {{#if @agendaitem.explanation }}
+    {{#if @agendaitem.comment }}
       <Auk::Modal::Footer @custom={{true}}>
         <Auk::Toolbar>
           <Auk::Toolbar::Group @position="left">
             <Auk::Toolbar::Item>
-              <pre data-test-agendaitem-titles-explanation class="auk-o-flex--inline auk-u-text-pre-line auk-u-italic">
-               {{t "remark-title"}}: {{@agendaitem.explanation}}
+              <pre data-test-agendaitem-titles-comment class="auk-o-flex--inline auk-u-text-pre-line auk-u-italic">
+               {{t "remark-title"}}: {{@agendaitem.comment}}
```
Comment (ValenberghsSven) [inline app/components/agenda/agendaitem/agendaitem-case/agendaitem-titles.hbs]: "remark-title" :sweat_smile:

Comment (sergiofenoll) [reply app/components/agenda/agendaitem/agendaitem-case/agendaitem-titles.hbs]: verdorie, volledig erover gelezen. merci :)

**Review by ValenberghsSven — APPROVED** [review]
Think we got everything covered

**Comment by sergiofenoll** [comment]
All green: http://kal-kastaar.s.redpencil.io:8080/job/kaleidos/job/custom_frontend_backend_combination/188/

**Pull request merged**